packages/orchestrator-ui-components/src/components/WFOStartTaskButtonComboBox/WFOStartTaskButtonComboBox.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+
+import { useTranslations } from 'next-intl';
+import { useQueryWithGraphql } from '../../hooks';
+import { GET_WORKFLOWS_FOR_DROPDOWN_LIST_GRAPHQL_QUERY } from '../../graphqlQueries/workflows/workflowsQueryForDropdownList';
+import {
+    WorkflowComboBoxOption,
+    WFOButtonComboBox,
+} from '../WFOButtonComboBox/WFOButtonComboBox';
+import { PATH_START_WORKFLOW } from '../WFOPageTemplate';
+
+export const WFOStartTaskButtonComboBox = () => {
+    const router = useRouter();
+    const t = useTranslations('common');
+
+    const { data } = useQueryWithGraphql(
+        GET_WORKFLOWS_FOR_DROPDOWN_LIST_GRAPHQL_QUERY,
+        {
+            // Avoiding pagination by fetching an unrealistic amount of items
+            first: 1000,
+            after: 0,
+            filterBy: [{ field: 'target', value: 'SYSTEM' }],
+        },
+        'taskWorkflows',
+    );
+
+    const productList: WorkflowComboBoxOption[] = data?.workflows.page.map(
```

Comment:
You also might want to sort the data before showing in the ComboBox

Reply:
I saw the sort in `WFOStartCreateWorkflowButtonComboBox`:I tried to delegate this to the backend, which seems to be the best choice. Not yet possible to sort on a subresult (products.name)....

So fixed it in JS (it feels a bit slow; you can already open the option box, whilst page is loading/sorting)